dnd.sty

```diff
@@ -8,7 +8,14 @@
 %
 % Prerequisite Packages
 %
-\RequirePackage[cm]{fullpage}     % Small margins
+% Set a different geometry with \newgeometry
```

Comment:
Can we stick to metric units? As far as I can tell, these would be the only uses of imperial.

Reply:
The inches are closer to a number that I could estimate with a ruler. I have misplaced my calipers, but I will see what I can do.

Reply:
So I at least was able to find my engineers triangular scale. The dimensions fit to inches a lot better than centimeters which I guess could make sense as the book was "made" in the states. I have a few sections that have fallen out of my PHB, so I will remeasure all the dimensions.